savoirfairelinux/sous-chef
# Fixed #636: replacement of Aaboffill_dev_636

Replaces PR #678.

## Fixes #636  by lingxiaoyang and aaboffill

### Changes proposed in this pull request:

On top of PR #678:
* Redefined certain on_delete values as PROTECT, SET_NULL and SET_DEFAULT (see the last commit).
* Rebased to dev
* Deleted the code concerning emergency contact (because it has already removed by #683).
* Added necessary migrations reflecting these model changes. Also fixed #670 in the same commit (e813dcaebba75436e556d14df58fdc7d44930421).
* Fixed a RemovedInDjango20 warning regarding django.urls


### Status


### How to verify this change

1. Compare travis build messages. Now, there should be no RemovedInDjango20Warning in build messages (ignore the vendor packages).

Current dev:
https://travis-ci.org/savoirfairelinux/sous-chef/builds/210664263

This PR:
https://travis-ci.org/savoirfairelinux/sous-chef/builds/210673532

2. Set up the environment, and run `manage.py makemigrations`. It should say "No changes detected". (Refs #670)

### Deployment notes and migration


### New translatable strings

none

### Additional notes

none
